### ISSUE TITLE
Builder failed when adding Node module to HelloWorld (last step)

### ISSUE THREAD

**Issue by fabioversolatto** [vtex/vtex-courses#195]
I did exatly the same step-by-step, by the way, the last step (when I have to add node builder) throws the error bellow.

I didn't add anything in my project...I'm just trying to do the helloworld sample.

20:13:46.216 - error: node@4.x builder failed to install dependencies through yarn (retries=0)
yarn errors:
graphql@16.0.1: The engine "node" is incompatible with this module. Expected version "^12.22.0 || ^14.16.0 || >=16.0.0". Got "12.16.1"
Found incompatible module. vtex.builder-hub@0.281.1
20:13:48.242 - error: node@4.x builder failed to install dependencies through yarn (retries=1)
yarn errors:
graphql@16.0.1: The engine "node" is incompatible with this module. Expected version "^12.22.0 || ^14.16.0 || >=16.0.0". Got "12.16.1"
Found incompatible module. vtex.builder-hub@0.281.1
20:13:50.555 - error: node@4.x builder failed to install dependencies through yarn (retries=2)
yarn errors:
graphql@16.0.1: The engine "node" is incompatible with this module. Expected version "^12.22.0 || ^14.16.0 || >=16.0.0". Got "12.16.1"
Found incompatible module. vtex.builder-hub@0.281.1
20:13:52.607 - error: node@4.x builder failed to install dependencies through yarn (retries=3)
yarn errors:
graphql@16.0.1: The engine "node" is incompatible with this module. Expected version "^12.22.0 || ^14.16.0 || >=16.0.0". Got "12.16.1"
Found incompatible module. vtex.builder-hub@0.281.1

**Comment by ViniciusNogueiraS** [comment]
Hi, I'm having the same problem... Have you made any progress?

**Comment by fabioversolatto** [comment]
Hi @ViniciusNogueiraS  ! not yet !

**Comment by zekivtex** [comment]
Same problem here

**Comment by zekivtex** [comment]
My partner Filadelfo Braz helped me. 
Solution:
- Let's go to the node folder
- Delete the @vtex/test-tools thread.
Problem: 
  In the yarn file we are importing graphql with * so we are getting the latest version which is causing the problem.

**Comment by fabioversolatto** [comment]
Hi @zekivtex  !!! thanks for your answer !
did it work?
because I did the same and the same error occurred :/

**Comment by ViniciusNogueiraS** [comment]
Hi, @zekivtex! I found a solution that worked for me.
Insert this into your `node/package.json`.
```
"resolutions": {
    "graphql": "14.6.0"
}
```

**Comment by fabioversolatto** [comment]
Hi guys !!

Thanks for answering !!!

I did what @ViniciusNogueiraS  told and the error bellow ocurred:
"graphql@16.1.0: The engine "node" is incompatible with this module. Expected version "^12.22.0 || ^14.16.0 || >=16.0.0". Got "12.16.1"
Found incompatible module. vtex.builder-hub@0.283.0"

my node version: 16.13.1
vtex cli version: vtex/3.8.1-beta.e606849e win32-x64 node-v12.12.0
(Could be this?)
